reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -321,278 +310,308 @@ class StaticHtmlReporter : Reporter() {
-                append("<ul>")
-                tabularScanRecord.evaluatorErrors?.let { evaluatorErrors ->
-                    append("<li><a href=\"#policy-violation-summary\">" +
-                            "Rule Violation Summary (${evaluatorErrors.size} violations)</a></li>")
-                }
-                if (numberOfErrors > 0) {
-                    append("<li><a href=\"#error-summary\">Error Summary ($numberOfErrors)</a></li>")
-                }
-                tabularScanRecord.projectDependencies.forEach { (project, projectTable) ->
-                    append("<li><a href=\"#${project.id}\">${project.id}")
-                    projectTable.exclude?.let { exclude ->
-                        append(" <div class=\"ort-reason\">Excluded: ${exclude.reason} - ${exclude.comment}</div>")
+                    if (reportTableModel.metadata.isNotEmpty()) {
+                        metadataTable(reportTableModel.metadata)
                     }
-                    append("</a></li>")
-                }
-                append("</ul>")
 
-                tabularScanRecord.evaluatorErrors?.let { evaluatorErrors ->
-                    append(createEvaluatorTable(evaluatorErrors))
-                }
+                    index(reportTableModel)
 
-                if (numberOfErrors > 0) {
-                    append(createErrorTable("Error Summary ($numberOfErrors)", tabularScanRecord.errorSummary,
-                            "error-summary"))
-                }
+                    reportTableModel.evaluatorErrors?.let {
+                        evaluatorTable(it)
+                    }
+
+                    if (reportTableModel.errorSummary.rows.count() > 0) {
+                        errorTable(reportTableModel.errorSummary)
+                    }
 
-                tabularScanRecord.projectDependencies.forEach { project, table ->
-                    append(createTable("${project.id} (${project.definitionFilePath})", project.vcsProcessed, table,
-                            project.id.toString()))
+                    reportTableModel.projectDependencies.forEach { project, table ->
+                        projectTable(project, table)
+                    }
                 }
             }
+        }
+
+        return document.serialize()
+    }
+
+    private fun DIV.metadataTable(metadata: Map<String, String>) {
+        h2 { +"Metadata" }
+        table("ort-report-metadata") {
+            tbody { metadata.forEach { key, value -> metadataRow(key, value) } }
+        }
+    }
+
+    private fun TBODY.metadataRow(key: String, value: String) {
+        tr {
+            td { +key }
+            td { if (value.isValidUrl()) a(value) { +value } else +value }
+        }
+    }
+
+    private fun DIV.index(reportTableModel: ReportTableModel) {
+        h2 { +"Index" }
+
+        ul {
+            reportTableModel.evaluatorErrors?.let {
+                li { a("#policy-violation-summary") { +"Rule Violation Summary (${it.size})" } }
```

Comment:
This drops the word "violations" compared to before, but I'm fine that that for consistency with "Error Summary".

Reply:
Done, see below.